tree-sitter-python/test/Test.hs

```diff
@@ -1,19 +1,26 @@
 {-# LANGUAGE DisambiguateRecordFields, OverloadedStrings, TypeApplications #-}
 module Main (main) where
 
+import           Control.Monad
+import           System.Path ((</>))
 import qualified System.Path as Path
+import           System.Path.Directory
 import           Test.Tasty
 import           TreeSitter.Python
 import qualified TreeSitter.Python.AST as Py
 import           TreeSitter.Test.Helpers
 import           TreeSitter.Unmarshal
 
 main :: IO ()
-main
-  =   readCorpusFiles (Path.relDir "./vendor/tree-sitter-python/corpus")
-  >>= traverse (testCorpus parse)
-  >>= defaultMain . tests
-  where parse = parseByteString @Py.Module @() tree_sitter_python
+main = do
+  dir <- getCurrentDirectory
+  when (Path.takeDirName dir == Just (Path.relDir "haskell-tree-sitter")) $
+    setCurrentDirectory (dir </> Path.relDir "tree-sitter-python")
+  let parse = parseByteString @Py.Module @() tree_sitter_python
+  readCorpusFiles (Path.relDir "./vendor/tree-sitter-python/corpus")
```

Comment:
What do you think about pushing this logic down into the test helpers (`readCorpusFiles`) so that other languages tests can avoid having to write this logic?

Reply:
Fair enough!